{{ cookiecutter.package_name }}/setup.py

```diff
@@ -82,7 +83,7 @@
 
 # Populate the dict of setup command overrides; this should be done before
 # invoking any other functionality from distutils since it can potentially
-# modify distutils' behavior.
+# modify distutils' behaviour.
```

Comment:
nice touch ;)

Reply:
I didn't even notice, I just corrected the spelling when emacs told me it was wrong lol